src/components/compact-button/styles.ts

```diff
@@ -0,0 +1,43 @@
+export default {
+  default: {
+    border: '1px solid #F5F5F5',
+    borderRadius: '2px',
+    backgroundColor: '#F5F5F5',
+    cursor: 'pointer',
+    boxShadow: 'none',
+    color: '#272727',
```

Comment:
please use colors specified in the theme at `src/theme/theme.ts`

Reply:
Done